README.md

```diff
@@ -70,7 +71,7 @@ GET http://localhost:8080/api/metadata/tagValues?
 ```http request
 GET http://localhost:8080/api/query?
   tenant=t-1
-  &metricName=cpu_idle
+  &metricKey=cpu_idle
```

Comment:
We have metric groups, names but why introduce yet another concept key?

Reply:
Maybe you could do:

`GET http://localhost:8080/api/query?tenant=t-1%metricGroup=cpu`
or
`GET http://localhost:8080/api/query?tenant=t-1%metricName=cpu_idle`

Having one par serving 2 purposes is not so clear.

Reply:
yes, I was thinking of that at first. but then we thought of changing it to metricKey and then considering metric group as of higher precedence than of metricName while finding for its complete data. 

@harageth , your inputs on this?

Reply:
metricName shouldn't change here. You can make another example with metricGroup